apps/cfp_review/forms.py

```diff
@@ -124,8 +124,7 @@ def update_proposal(self, proposal):
 
         if self.needs_laptop.raw_data:
             proposal.needs_laptop = self.needs_laptop.data
-        if self.may_record.raw_data:
-            proposal.may_record = self.may_record.data
+        proposal.may_record = self.may_record.data
```

Comment:
Would this also apply to needs_laptop?

Reply:
No; needs_laptop is a SelectField between 0/1, rather than a checkbox.

Maybe this should be a SelectField too?